cli/src/config/templates.toml

```diff
@@ -281,9 +288,39 @@ separate(" ",
         if(commit.git_head(), label("git_head", "git_head()")),
         format_short_commit_id(commit.commit_id()),
         if(commit.conflict(), label("conflict", "conflict")),
+        if(config("ui.show-cryptographic-signatures").as_boolean(),
+          format_short_cryptographic_signature(commit.signature())),
       )
 '''
 
+# signature verification is slow, disable by default
+'format_detailed_cryptographic_signature(signature)' = '''
+if(signature,
+  separate(" ",
+    format_cryptographic_signature_status(signature),
```

Comment:
nit: might be better to use less fancy output in detailed template. just print `"[" ++ signature.status() ++ "]"`?

Reply:
+1, the detailed template should be more of a plain english type of thing probably

"Good <backend if/when we ever re-add the backend field> signature by <display>"